index.js

```diff
@@ -315,6 +438,62 @@ class EmeEncryptionSchemePolyfillMediaKeySystemAccess {
   }
 }
 
+/**
+ * Guess the supported encryption scheme for the key system.
+ *
+ * @param {string} keySystem The key system ID.
+ * @return {?string} A guess at the encryption scheme this key system
+ *   supports.
+ */
+function guessSupportedScheme(keySystem) {
+  if (keySystem.startsWith('com.widevine')) {
+    return 'cenc';
+  } else if (keySystem.startsWith('com.microsoft')) {
+    return 'cenc';
+  } else if (keySystem.startsWith('com.adobe')) {
+    return 'cenc';
+  } else if (keySystem.startsWith('org.w3')) {
+    return 'cenc';
+  } else if (keySystem.startsWith('com.apple')) {
+    return 'cbcs-1-9';
+  }
+
+  // We don't have this key system in our map!
+
+  // Log a warning.  The only way the request will succeed now is if the
+  // app doesn't specify an encryption scheme in their own configs.
+  // Use bracket notation to keep this from being stripped from the build.
+  console['warn']('EmeEncryptionSchemePolyfill: Unknown key system:',
+      keySystem, 'Please contribute!');
+
+  return null;
+}
+
+/**
+ * @param {?MediaKeySystemAccess} mediaKeySystemAccess A native
```

Comment:
This is nullable but you don't check for null before using.

Reply:
> This is nullable but you don't check for null before using.

It's nullable because the field on the upstream object is nullable.  Because I'm not depending on goog.asserts, there's no way for me to assert to the compiler that it's non-null at this time.  But it is.  You only get null on the upstream field if supported is false or the original query was for clear content.  We have already checked for those things.